### PR TITLE
Update `README.md`

### PR DESCRIPTION
**Pull request by mogasergiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
+$ make
+```
+
+Run the application:
+```
+sudo qemu-system-x86_64 -fsdev local,id=myid,path=$(pwd)/fs0,security_model=none \
+                        -device virtio-9p-pci,fsdev=myid,mount_tag=rootfs,disable-modern=on,disable-legacy=off \
+                        -kernel "build/app-sqlite_kvm-x86_64" \
+                        -enable-kvm
+                        -nographic
```
Comment (gabrielmocanu) [inline README.md]:
```suggestion
                        -kernel "build/app-sqlite_kvm-x86_64" \
                        -enable-kvm \
                        -nographic
```

**Review by gabrielmocanu — CHANGES_REQUESTED** [review]

**Comment by gabrielmocanu** [comment]
@mogasergiu 

It worked for you to run the application using kraft?
I got an error at the mounting part with kraft, but it's working in the classic way.

File: kraft.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ unikraft:
     - CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS=y
     - CONFIG_LIBVFSCORE_ROOTFS_9PFS=y
     - CONFIG_LIBUKLIBPARAM=y
-    - CONFIG_LIBVFSCORE_ROOTDEV="fs0"
```
Comment (gabrielmocanu) [inline kraft.yaml]: @mogasergiu 

Is this change necessary? I think it's better to left with fs0 because otherwise it won't work using kraft.
Kraft is using qemu-guest to start the application and qemu-guest have support only for tag as `fs0`, `fs1`, etc.
Otherwise it would need an extra step adding, to change in `menuconfig` from `rootfs`  to `fs0`.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,50 @@
...
+
+Build the application:
+```
+$ make
+```
+
+Run the application:
+```
+sudo qemu-system-x86_64 -fsdev local,id=myid,path=$(pwd)/fs0,security_model=none \
+                        -device virtio-9p-pci,fsdev=myid,mount_tag=rootfs,disable-modern=on,disable-legacy=off \
```
Comment (gabrielmocanu) [inline README.md]:
```suggestion
                        -device virtio-9p-pci,fsdev=myid,mount_tag=fs0,disable-modern=on,disable-legacy=off \
```

If we do the change in `kraft.yaml` file for mounting tag to be `fs0` I think we should update also here because most of the people will build with kraft and then use `kraft run` or this command to run the application and we need to have the same mounting tag.

**Review by gabrielmocanu — CHANGES_REQUESTED** [review]

**Review by gabrielmocanu — APPROVED** [review]
Looks good now.

Reviewed-by: Gabi Mocanu <gabi.mocanu98@gmail.com>

**Review by razvand — APPROVED** [review]
Approved-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>
